src/Umbraco.Infrastructure/PropertyEditors/TagsPropertyEditor.cs

```diff
@@ -107,19 +106,41 @@ internal class TagPropertyValueEditor : DataValueEditor, IDataValueTags
             IDataTypeService dataTypeService)
             : base(localizedTextService, shortStringHelper, jsonSerializer, ioHelper, attribute)
         {
+            _jsonSerializer = jsonSerializer;
             _dataTypeService = dataTypeService;
         }
 
         /// <inheritdoc />
         public IEnumerable<ITag> GetTags(object? value, object? dataTypeConfiguration, int? languageId)
         {
-            var strValue = value?.ToString();
-            if (string.IsNullOrWhiteSpace(strValue)) return Enumerable.Empty<ITag>();
+            TagConfiguration tagConfiguration = ConfigurationEditor.ConfigurationAs<TagConfiguration>(dataTypeConfiguration) ?? new TagConfiguration();
+
+            var tags = ParseTags(value, tagConfiguration);
+            if (tags.Any() == false)
```

Comment:
More pattern matching 😁 
```suggestion
            if (tags.Any() is false)
```